src/navigation/MainNavigator.tsx

```diff
@@ -47,6 +51,14 @@ const MainNavigator: FunctionComponent = () => {
                 headerShown: false,
               }}
             />
+            <Stack.Screen
+              name={Stacks.Callback}
+              component={CallbackStack}
+              options={{
+                ...Headers.headerScreenOptions,
```

Comment:
At some point we probably want to remove this and add the header within the `Callback` stack, like we do in `Settings`.